### ISSUE TITLE
Some subdomains are not matched with Base domain detection option

### ISSUE THREAD

**Issue by obvionaoe** [bitwarden/browser#2304]
### Steps To Reproduce

1. Create a login with base domain match for `https://gov.pt`
2. Go to 'gov.pt'
3. See recommended login for that website
4. Go to https://www.acesso.gov.pt and check that it's missing
5. Go to https://www.portaldasfinancas.gov.pt and check that it's also missing

### Expected Result

The base domain match option should match all subdomain of `gov.pt`

Such as:

- portaldasfinancas.gov.pt
- acesso.gov.pt
- ...

### Actual Result

The login is only recommended for `gov.pt`

### Operating System

Linux

### Operating System Version

Manjaro

### Web Browser

Firefox

### Browser Version

95.0.2-1

### Build Version

1.55.0

**Comment by zefilipe** [comment]
Hi, 

Open BitWarden -> logins !
Its shows the login for gov.pt, click in view -> auto fill and save, and now edit, on the url settings select host.

This maybe solve this problem !
I use subdomains in my work, and for me its work !

If you need create multiple logins (example: one for acesso.gov.pt and other for portaldasfinancas.gov.pt
Regards

**Comment by MartinX3** [comment]
Maybe it's the same issue:
The firefox extension lists every credential regardless of the subdomain.
All of my credentials for
-  base.org
- x.base.org
- y.base.org
- z.base.org
are listed on each side, which doesn't make any sense.

**Comment by obvionaoe** [comment]
I agree, but I still believe this should be fixed or if it is the intended functionality at least the documentation should be clarified

**Comment by MartinX3** [comment]
Yes, it must be a bug

**Comment by jorisvanherp** [comment]
I have the same problem with herokuapp.com domains.

**Comment by djsmith85** [comment]
The originally reported issue might have to do with some TLD's (pt/gov.pt) not being recognized as valid domains. This should get addressed with the linked PR.

In general, if you are expecting to match with a specific sub-domain (i.e. "**neo4j.sync.** auth0.com") and not just the domain (**auth0.com**) then you might need to adjust your URI detection to `host` for those items.

Here's a link to our documentation around URI match detection: https://bitwarden.com/help/uri-match-detection/
